.circleci/continue_config.yml

```diff
@@ -105,9 +105,6 @@ jobs:
       - run:
           name: Checkout LFS blobs
           command: git lfs pull
-      - run:
```

Comment:
One nit is to add comment on the now-private material usage for tracking. Or can we use alternative icon?